README.md

```diff
@@ -166,11 +166,15 @@ Yes, in fact - it has nothing to do with it. What it does is just linking static
 Sure you can! Try doing something like this in your package.json:
 ```json
 ...
-"scripts": {
-  "postinstall": "rnpm link"
+"dependencies": {
+  "rnpm": "^1.4.1"
 },
+"scripts": { 
+  "postinstall": "cd ../.. && rnpm link <name>"
+}
 ...
 ```
+Note that the `cd ../..` in the postinstall script is needed to go up to the working directory of the parent project that is to be linked with your plugin.
```

Comment:
In 99% cases there's only one package.json and rnpm works just fine from the root folder. Are you having a different use-case?